scripts/workers/code_upload_submission_worker.py

```diff
@@ -567,6 +589,19 @@ def update_failed_jobs_and_send_logs(
                             logger.exception(
                                 "Exception while reading Job logs {}".format(e)
                             )
+                    elif is_submission_container_executed(
+                        container_name, container_state, container_state_map
+                    ):
```

Comment:
Also add a one line comment describing the check

Reply:
@Ram81 Docstring is added in the function heading, should I add that here as well?